packages/shared/providers.yaml

```diff
@@ -1759,6 +1793,8 @@ contentstack:
     authorization_url: https://${connectionConfig.subdomain}.contentstack.com/apps/${connectionConfig.appId}/authorize
     token_url: https://${connectionConfig.subdomain}.contentstack.com/apps-api/apps/token
     docs: https://docs.nango.dev/integrations/all/contentstack
+    proxy:
+        base_url: https://${connectionConfig.apiBaseUrl}
```

Comment:
could we always put `.contentstack.com`?

Reply:
This may also hugely vary, for Knowledge vault api we can have `https://gcp-na-ai.contentstack.com/brand-kits`, while for brand kit management api we can have `https://gcp-na-brand-kits-api.contentstack.com`